lapis/drone.py

```diff
@@ -47,9 +48,16 @@ def __init__(self, scheduler, pool_resources: dict,
 
     @property
     def theoretical_available_resources(self):
+        levels = self.resources.levels
         return {
-            key: self.pool_resources[key] - self.resources[key]
-            for key in self.pool_resources
+            key: getattr(levels, key) for key in self.pool_resources
+        }
+
+    @property
+    def available_resources(self):
+        levels = self.used_resources.levels
+        return {
+            key: getattr(levels, key) for key in self.pool_resources
```

Comment:
You may want to exploit that ``ResourceLevels`` are an iterable over ``field, value`` pairs. Try if ``dict(self.used_resources.levels)`` works for you.

Reply:
Seems to work, thanks.